File: C_Programming/Competitive_Programming/CP_016.c
//////////////////////////////////////////////////////////////////
//
//  File Name    : Multiplication_Of_Factors
//
//  Description  : This program accepts a number from the user
//                 and returns the multiplication of all its
//                 factors excluding the number itself.
//
//  Example:
//      Input  : 12
//      Output : 144
//
//      Input  : 13
//      Output : 1
//
//////////////////////////////////////////////////////////////////

#include<stdio.h>

int MultFact(int iNo)
{
    int i = 0;
    int iMult = 1;
    
    if(iNo <= 0)
    {
        iNo = -iNo;
    }

    for(i = 1; i < iNo; i++)
    {
        if(iNo % i == 0)
        {
            iMult = iMult * i;
        }
    }
    return iMult;
}

int main()
{
    int iValue = 0;
    int iRet = 0;

    printf("Enter Number : \n");
    scanf("%d", &iValue);

    iRet = MultFact(iValue);

    printf("%d", iRet);

    return 0;
}

File: Java_Programming/Competitive_Programming/CP_016.java
//////////////////////////////////////////////////////////////////
//
//  File Name    : Sum_Of_Even_Numbers
//
//  Description  : This program accepts a number and
//                 calculates the sum of all even
//                 numbers from 1 up to that number.
//
//  Example:
//      Input  : 10
//      Output : Sum of all Even Numbers is : 30
//
//////////////////////////////////////////////////////////////////

class Logic
{
    void SumEvenNumbers(int iNum)
    {
        int iCnt = 0, iSum = 0;

        for(iCnt = 1; iCnt <= iNum; iCnt++)
        {
            if(iCnt % 2 == 0)
            {
                iSum = iSum + iCnt;
            }
        }
        System.out.println("Sum of all Even Numbers is : " + iSum);
    }
}

class CP_016
{
    public static void main(String A[])
    {
        Logic lobj = new Logic();

        lobj.SumEvenNumbers(10);
    }
}